platform/src/test/java/org/hillview/test/CountSketchTest.java

```diff
@@ -0,0 +1,111 @@
+package org.hillview.test;
+
+import org.hillview.dataset.api.Pair;
+import org.hillview.sketches.*;
+import org.hillview.table.Table;
+import org.hillview.table.api.ITable;
+import org.hillview.table.rows.RowSnapshot;
+import org.hillview.utils.TestTables;
+import org.junit.Test;
+
+public class CountSketchTest {
+    public boolean toPrint = true;
+
+    private void timeCreate(ExactCountSketch cs, ITable table) {
+        long oldStart = System.nanoTime();
+        FreqKList freqK = cs.create(table);
+        long oldEnd = System.nanoTime();
+        System.out.printf("Time: %d\n", oldEnd - oldStart);
```

Comment:
would you like to guard this with a verbose flag?
Maybe we should move the verbose flag in the BaseTest class.

Reply:
That was only for comparing two implementations. The routine is gone now.